Makefile-tests.am

```diff
@@ -121,6 +121,7 @@ _installed_or_uninstalled_test_scripts += \
 	tests/test-refs-collections.sh \
 	tests/test-remote-add-collections.sh \
 	tests/test-summary-collections.sh \
+	tests/test-pull-collections.sh \
```

Comment:
Want me to squash all three commits into one? The test is using features added in both previous commits.